### ISSUE TITLE
pretraining hyperparamters

### ISSUE THREAD

**Issue by junchen14** [ArrowLuo/CLIP4Clip#6]
when you pretrain on howto100m dataset, do you use the same pretrain hyperparamters as you used in  training on msvrtt and msvd small datasets (especially the learning rate).  
did you also test other hyperparamters? if so, it would be good to share some experiences. 
thanks

**Comment by ArrowLuo** [comment]
Hi, Yes, the hyperparameters are the same as the finetune on the msvrtt except for the batch size. The learning rate is also 1e-7. We have not tested other hyperparameters yet. Because the pretrain is time-consuming.

**Comment by ArrowLuo** [comment]
I think 1e-4 is ok, or you can set it smaller.

**Comment by junchen14** [comment]
ok got it.  thanks very much for your quick response
